ddl/ddl_api.go

```diff
@@ -1278,38 +1279,35 @@ func (d *ddl) CreateView(ctx sessionctx.Context, s *ast.CreateViewStmt) (err err
 	return d.callHookOnChanged(err)
 }
 
-func buildViewInfoWithTableColumns(ctx sessionctx.Context, s *ast.CreateViewStmt) (*model.ViewInfo, []*table.Column) {
+func buildViewInfoWithTableColumns(ctx sessionctx.Context, s *ast.CreateViewStmt) (*model.ViewInfo, []*table.Column, error) {
+	singleQuoteFlag := format.RestoreStringSingleQuotes | format.RestoreKeyWordUppercase | format.RestoreNameBackQuotes
+	doubleQuoteFlag := format.RestoreStringDoubleQuotes | format.RestoreKeyWordUppercase | format.RestoreNameDoubleQuotes
```

Comment:
Have you confirmed this, when will `format.RestoreStringDoubleQuotes` be used?

Reply:
`format.RestoreStringDoubleQuotes` will use char `"` as bracket